### ISSUE TITLE
Free resource and you are obfuscateing the .js files.. BRUh

### ISSUE THREAD

**Issue by ImpulseFPS** [Kocadede/kcdd-banking#1]
deobfuscate the .js files cuz some of the code in js doesn't respond to Nui callbacks

**Comment by WooAf** [comment]
Hahahahha

**Comment by ghost** [comment]
bruh xD

**Comment by FokkenClown** [comment]
yeah would also love the deobfuscated version of the js file as there are some issues with this version that i would love to fix and changes that are required. 

Like transfer should show on your history as transfer with - not deposit, and on recipeint as transfer with +

also need to integrate this with our phone transfers.

Currently working on creating my own version of this.
But would be so much simpilar to just have all the code of a free file, or an option to purchase the file with all code unlocked
